src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceDefinition.scala

```diff
@@ -236,33 +236,11 @@ object InstanceDefinition {
       t: TypePropertyDefinition
   ): Either[DecodingFailure, InstancePropertyValue] =
     t.`type` match {
-      case _: PropertyType.DirectNodeRelationProperty =>
+      case PropertyType.DirectNodeRelationProperty(_, _, Some(false)) |
+          PropertyType.DirectNodeRelationProperty(_, _, None) =>
         propValue
           .as[Option[DirectRelationReference]]
           .map(InstancePropertyValue.ViewDirectNodeRelation.apply)
-          .orElse {
-            // TODO: Remove this once this is fixed
-            // https://cognitedata.slack.com/archives/C031G8Y19HP/p1676895220201609
```

Comment:
tthread looks resolved, so hopefully safe to remove :crossed_fingers:

Reply:
yeah I think it was also somewhat covered by tests, because tests were posting and getting using this. So if the "get" endpoint was still returning a list and it needed this to function some tests would fail.